packages/orm/src/changes.ts

```diff
@@ -17,8 +17,6 @@ export interface ManyToOneFieldStatus<T extends Entity> extends FieldStatus<IdOf
   originalEntity: Promise<T | undefined>;
 }
 
-type ExcludeNever<T> = Pick<T, { [P in keyof T]: T[P] extends never ? never : P }[keyof T]>;
```

Comment:
The key mapping feature of TS 4.1 (i.e. mapping `foo` -> `fooChanged`) also means `ExcludeNever` is not necessary anymore b/c if you do:

```
[P in keyof T as ...some condition... extends never ? never : K]: ...
```

Any keys that the `as` evaluates to `never` will just be dropped from the type.

A little verbose, but neat. I rewrote some of the `LoadHint` / `Loadable` things to use this.